apps/api/e2e/tests/portfolio-snapshots.test.ts

```diff
@@ -272,164 +270,26 @@ describe("Portfolio Snapshots", () => {
-      const priceReuseMessages = logMessages.filter(
-        (msg) =>
-          msg.includes("[CompetitionManager]") &&
-          msg.includes("Using fresh price") &&
-          msg.includes("from DB"),
-      );
-
-      console.log(
-        `[Test] Found ${priceReuseMessages.length} instances of price reuse`,
-      );
-
-      // Look for DB hit messages to confirm we're finding price records
-      const dbHitMessages = logMessages.filter(
-        (msg) =>
-          msg.includes("[CompetitionManager]") &&
-          msg.includes("Price lookup stats") &&
-          msg.includes("DB hits"),
-      );
-
-      if (dbHitMessages.length > 0) {
-        console.log(`[Test] DB hit stats: ${dbHitMessages[0]}`);
-      }
-
-      // For debugging, relaxing the assertion and printing more info instead
-      if (priceReuseMessages.length === 0) {
-        console.log(
-          `[Test] WARNING: No price reuse detected. This could be because:`,
-        );
-        console.log(
-          `[Test] 1. The price freshness threshold might not be working`,
-        );
-        console.log(`[Test] 2. No prices were found in the database`);
-        console.log(
-          `[Test] 3. The log message format differs from what we're looking for`,
-        );
-
-        // Check if we found any tokens in DB but didn't reuse them
-        const specifcChainMessages = logMessages.filter(
-          (msg) =>
-            msg.includes("[CompetitionManager]") &&
-            msg.includes("Using specific chain info from DB"),
-        );
-
-        if (specifcChainMessages.length > 0) {
-          console.log(
-            `[Test] Found ${specifcChainMessages.length} messages about using chain info from DB but not price`,
-          );
-          console.log(`[Test] Example: ${specifcChainMessages[0]}`);
-        }
-      }
-
-      // Change to a softer assertion for now - we're debugging
-      // We'll log a warning instead of failing the test
-      if (priceReuseMessages.length === 0) {
-        console.warn(
-          `[Test] Expected to find price reuse messages but none were found`,
-        );
-      }
-
-      // Extract the overall stats
-      const statsMessage = logMessages.find((msg) =>
-        msg.includes("Reused existing prices:"),
-      );
-      if (statsMessage) {
-        console.log(`[Test] Stats: ${statsMessage}`);
-
-        // Extract the reuse percentage from the log message
-        const reusePercentage = parseFloat(
-          statsMessage?.match(/\((\d+\.\d+)%\)/)?.[1] || "0",
-        );
-
-        console.log(`[Test] Reuse percentage: ${reusePercentage}%`);
-      } else {
-        console.log(`[Test] No reuse statistics found in logs`);
-      }
-    } finally {
-      // Restore the original console.log
-      console.log = originalConsoleLog;
-    }
+    // call again before cache expires
+    await priceTracker.getPrice(usdcTokenAddress);
```

Comment:
This test amounted to spying on console.log and never really asserting. It was also wrapped in a try catch finally.  I simplified and just use a spy to make sure the cache is hit.  
Note, I needed to remove the private decorator from the `getCachedPrice` method in order to spy on the method.